src/queue.js

```diff
@@ -25,17 +25,26 @@ export default class Queue {
         this.tail = node
       }
 
-      this.length += 1
-
+      this.size += 1
       return node
   }
 
   dequeue() {
-    
+    if (this.size === 0) return null
```

Comment:
I recommend avoiding inline control flow, and explicitly using braces to define the associated block.  This increases readability, makes the code more explicit, and avoids potential dangling if/else bugs, and increases ease of maintenance.